crypto/conf/conf_lcl.h

```diff
@@ -7,5 +7,26 @@
  * https://www.openssl.org/source/license.html
  */
 
-void conf_add_ssl_module(void);
```

Comment:
I would rather keep this one in place, separate from what you add.

Reply:
Why?  I moved the declaration to be with all the other function declarations in the header file.

Reply:
Because configuration modules are separate things.

Reply:
If this gets another review and that is the only thing holding back merging, then I will make the change.

Reply:
I added some comments, is that good enough?

Reply:
That works